phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexScrutinyTool.java

```diff
@@ -114,18 +115,27 @@
             "If specified, uses Tenant connection for tenant view index scrutiny (optional)");
     public static final String INDEX_JOB_NAME_TEMPLATE = "PHOENIX_SCRUTINY_[%s]_[%s]";
 
+    @Inject
+    Class<IndexScrutinyMapperForTest> mapperClass = null;
+
+    public IndexScrutinyTool(Class<IndexScrutinyMapperForTest> indexScrutinyMapperForTestClass) {
```

Comment:
I want to use Class<?> instead of providing Test class . However, while invoking setMapperClass with this attribute, it complains that setMapperClass cant be applied to capture<?> Any other ideas? @gjacoby126